### DISCUSSION TITLE
aws lambda cleanup

### DISCUSSION THREAD

**Discussion by BartoszZabuski** [artilleryio/artillery#1960] [Artillery]
hello everyone,

Having issues running artillery via lambda. All seems to work fine, test runs, lambda ends but my console is being stuck with a spinning icon. No test summary and lambda staying in AWS.

The only error I see is below one at the beginning of the lambda run:
```2023-07-03T17:10:19.194Z	8cd537e5-3a2a-4293-8d54-431ee95f60f1	ERROR	(node:25) [MODULE_NOT_FOUND] Error Plugin: artillery: Cannot find module '@artilleryio/platform-fargate'
Require stack:
- /var/task/node_modules/artillery/lib/cmds/run-fargate.js
- /var/task/node_modules/artillery/node_modules/@oclif/config/lib/plugin.js
- /var/task/node_modules/artillery/node_modules/@oclif/config/lib/config.js
- /var/task/node_modules/artillery/node_modules/@oclif/config/lib/index.js
- /var/task/node_modules/artillery/node_modules/@oclif/command/lib/command.js
- /var/task/node_modules/artillery/node_modules/@oclif/command/lib/index.js
- /var/task/node_modules/artillery/bin/run
module: @oclif/config@1.18.11
task: toCached
plugin: artillery
root: /var/task/node_modules/artillery
See more details with DEBUG=*
(Use `node --trace-warnings ...` to show where the warning was created)
```

Also anyone knows how to effectively clean up all resources after failed runs like that?

**Comment by hassy** [comment]
Hi @BartoszZabuski 👋 that looks like a possible bug - is it happening for you consistently? and is that on the latest version of Artillery?

To clean up manually, you'd need to delete the AWS Lambda function that gets created by the test, and an SQS queue that gets created in the same region.
